devtools/db_import/db_import/local_storage.py

```diff
@@ -0,0 +1,51 @@
+## Copyright 2023 The OpenXLA Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+"""This API mimics the Google Cloud Storage Python APIs.
+   Instead of accessing files in GCP, it's backed by the local
+   filesystem. Only reading and listing files is supported.
+"""
+
+import os
+import pathlib
+
+
+class Blob:
+
+  def __init__(self, file: pathlib.Path, bucket):
+    self.file: pathlib.Path = file
+    self.bucket = bucket
+
+  @property
+  def name(self):
+    return os.path.relpath(self.file, self.bucket.path)
```

Comment:
I guess you can use `self.file.relative_to(self.bucket.path)` (https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.relative_to)

Reply:
Awesome. Yes. Thank you.